backend/Squirrel.ConsoleApp.BL/Squirrel.ConsoleApp.BL.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+      <PackageReference Include="Npgsql" Version="7.0.4" />
```

Comment:
i guess this package version is for .NET 7, we use .NET 6

Reply:
yea, I'll change version

Reply:
no need to change the version, the package is from .NET 6